### PR TITLE
Some extra characters that we'll be stuck without

### PR DESCRIPTION
**Pull request by kaithar**
Without -, = and . we're going to be pretty stuck.  The / makes things much easier, as does the >.  

Escape is in there for Vi, though Emacs would be pretty bad since we have no control combinations

### CODE REVIEW

File: map.json [hunk 1]
```diff
@@ -51,6 +51,14 @@
   "Y": "sendkey shift-y",
   "z": "sendkey z",
   "Z": "sendkey shift-z",
+  "-": "sendkey minus",
+  ".": "sendkey dot",
+  "/": "sendkey slash",
+  ";": "sendkey semicolon",
+  ":": "sendkey shift-semicolon",
+  "=": "sendkey equal",
+  ">": "sendkey greater",
```
Comment (kaithar) [inline map.json]: I should note that I'm guessing on this one based on the keymap similarity

**Comment by kaithar** [comment]
Ok, that should be enough for people to actually manage console use. :)

**Comment by kaithar** [comment]
Now that should be enough :shipit:

**Comment by dlaynes** [comment]
Hi, guys. I think a "wait" command would be useful to counter "ctrl+c" spam during an installation.
Cheers.

**Pull request merged**